docs/guides/tools.md

```diff
@@ -46,6 +46,7 @@ Tools are installed into temporary, isolated environments when using `uvx`.
     If you are running a tool in a [_project_](../concepts/projects.md) and the tool requires that your project is installed,
     e.g., when using `pytest` or `mypy`, you'll want to use [`uv run`](./projects.md#running-commands) instead of `uvx`.
     Otherwise, the tool will be run in a virtual environment that is isolated from your project.
+    However you can still use `uvx`, but it's not recommended.
```

Comment:
Does `pytest` actually work here? You're in a project and running `uvx pytest` and your tests import your project? Are you using a flat project structure?

Reply:
well [here is the project I used](https://github.com/ZeyadMoustafaKamal/django-rest-admin) clone it and see if this is the same thing you were talking about.

Reply:
Yeah this is because you're using a flat project structure, so the project can be imported _without_ being installed (vs e.g., using `src/<module>` for your code)

Reply:
I can try to improve the docs here.

Reply:
fine. and thanks a lot for your hard work.

Reply:
Also I think we can close this now or what you think